README.md

```diff
@@ -63,178 +72,366 @@ bambuAnnotations <- prepareAnnotations(gtf.file)
 
 ```rscript
-bambu(reads = test.bam, annotations = txdb, genome = fa.file, discovery = FALSE)
+se.discoveryOnly <- bambu(reads = test.bam, annotations = gtf.file, genome = fa.file, quant = FALSE)
 ```
 
-**Large sample number/ limited memory**     
-For larger sample numbers we recommend to write the processed data to a file:
+**Quantification of annotated transcripts and genes only (no transcript/gene discovery)**
+
+If you are only interested in quantifying transcripts, the discovery module of bambu can be skipped by setting discovery to FALSE.
 
 ```rscript
-bambu(reads = test.bam, rcOutDir = "./bambu/", annotations = bambuAnnotations, genome = fa.file)
+se.quantOnly <- bambu(reads = test.bam, annotations = gtf.file, genome = fa.file, discovery = FALSE)
+```
+### Using precalculated annotation objects
+Depending on the size of your reference annotations the prepareAnnotations() step may take a few minutes. You can also use precalculated annotations and if you plan to run bambu more frequently with the same annotations, we recommend to save the bambuAnnotations object.
+The bambuAnnotation object can be calculated from a .gtf file:
+```rscript
+annotations <- prepareAnnotation(gtf.file)
+```
+From TxDb object
+```rscript
+annotations <- prepareAnnotations(txdb)
+```
+Save the object
+```rscript
+saveRDS(annotations, ”/path/to/annotations.rds” )
 ```
+This object can then be used instead of a path to your reference annotations to the annotations argument.
+```rscript
+annotations <- readRDS("/path/to/annotations.rds")
+bambu(reads <- test.bam, annotations = annotations, genome = fa.file)
+```
+### Running multiple samples
 
-For very large samples (>100 million reads) where memory is limiting we recommend running Bambu in lowMemory mode:
+If you have multiple replicates for a sample, or plan to do comparative analysis between conditions, it may be beneficial to run all samples together instead of individually. This can be done by providing a vector of paths to all the bam files you want to analyze together.
 
 ```rscript
-bambu(reads = test.bam, annotations = bambuAnnotations, genome = fa.file, lowMemory = TRUE)
-```
 
----
+se.multiSample <- bambu(reads = c(test1.bam, test2.bam, test3.bam), annotations = gtf.file, genome = fa.file)
+```
 
+The advantage of running samples together include:
+Novel transcripts that are identified in multiple samples are assigned unified IDs, enabling comparative analysis between different samples. This is especially important for downstream differential expression analysis when looking at novel transcripts. Running multiple samples can be multithreaded (see ncore). While running multiple samples, By default, *bambu* will train a model separately on each sample and score novel transcripts in each sample separately.
```

Comment:
could be great if ncore put a link that goes to Bambu argument.

Reply:
I don't know how to do this

Reply:
I made it link to arguments but I don't know if I can make it more precise

Reply:
I think this is sufficient.